src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/02 Advanced Examples.md

```diff
@@ -3,11 +3,19 @@ title: 'Advanced Examples'
 excerpt: 'Advanced Examples using the k6 Scenario API - Using multiple scenarios, different environment variables and tags per scenario.'
 ---
 
-## Using multiple scenarios
+You can use multiple scenarios in one script, and these scenarios can be run in sequence or in parallel.
+Some ways that you can combine scenarios include the following:
+- Sequence workload start times
+- Add per-scenario tags and environment variables
+- Make scenario-specific thresholds.
+- Run scenarios in different VU [lifecycle functions](/using-k6/test-lifecycle).
```

Comment:
I don't understand what you mean here, sorry :confused: You can't have different `setup()` or `teardown()` per scenario (see https://github.com/grafana/k6/issues/1638, though it will probably be solved by actually implementing https://github.com/grafana/k6/issues/1342)

Reply:
I'm trying to communicate what is done in this example, by the way.
Different scenarios run different VU logic.

```js
export function contacts() {
  http.get('https://test.k6.io/contacts.php', {
    tags: { my_custom_tag: 'contacts' },
  });
}

export function news() {
  http.get('https://test.k6.io/news.php', { tags: { my_custom_tag: 'news' } });
}
```